apps/backend/.env-template

```diff
@@ -0,0 +1,11 @@
+# The password for the keystore and the SSL certificate.
+# type:     string
+# optional (required when running locally since SSL is enforced in that case.)
+# default:
+KEYSTORE_PASSWORD="<keystore-password>"
+
+# The mode in wich the application should run. Determines which configuration is picked up.
+# type:     "prod" | "dev"
+# optional
+# default: "prod"
+MODE=dev
```

Comment:
Again, can be left to `MODE` but it would be better to rename this variable to `KTOR_ENV`.